### PR TITLE
schedulers: add some metrics for hotreigon

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve? 
- add some metrics and log to observe hot region scheduling.
- Reduce the frequency of hotspot scheduling.

### What is changed and how it works?
add metrics.

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has configuration change
 - Has HTTP API interfaces change (Don't forget to [update API document](https://github.com/pingcap/pd/blob/master/docs/development.md#updating-api-documentation))
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### CODE REVIEW

File: server/statistics/store.go [hunk 1]
```diff
@@ -19,6 +19,9 @@ import (
 
 	"github.com/pingcap/kvproto/pkg/pdpb"
 	"github.com/pingcap/pd/v4/server/core"
+
```
Comment (lhy1024) [inline server/statistics/store.go]: unexpected empty line

File: server/cluster/coordinator.go [hunk 2]
```diff
@@ -402,6 +402,7 @@ func (c *coordinator) collectHotSpotMetrics() {
 		stat, ok := status.AsPeer[storeID]
 		if ok {
 			hotSpotStatusGauge.WithLabelValues(storeAddress, storeLabel, "total_written_bytes_as_peer").Set(stat.TotalBytesRate)
+			hotSpotStatusGauge.WithLabelValues(storeAddress, storeLabel, "total_written_keys_as_peer").Set(stat.TotalBytesRate)
```
Comment (rleungx) [inline server/cluster/coordinator.go]: Does it need to set 0 in else?

**Pull request merged**